src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj

```
@@ -15,12 +15,13 @@ Microsoft.Data.Sqlite.SqliteException
 Microsoft.Data.Sqlite.SqliteFactory
 Microsoft.Data.Sqlite.SqliteParameter
 Microsoft.Data.Sqlite.SqliteTransaction</Description>
-    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework);netstandard2.0</TargetFrameworks>
```

Why do we need to target 8.0? Can we just [synthesize some attributes](https://github.com/dotnet/efcore/blob/main/src/Microsoft.Data.Sqlite.Core/Utilities/AllowNullAttribute.cs) like we do for nullable on netstandard2.0? If it's really needed, we might want to consider multi-targeting everything inbtween so it's not a breaking change:
```suggestion
    <TargetFrameworks>$(DefaultNetCoreTargetFramework);netstandard2.0;net6.0;net7.0</TargetFrameworks>
```

---

To clarify, the breaking change is that the netstandard2.0 assembly will now be used on net6.0 which will remove some features like savepoints.

---

Given that M.D.Sqlite multitargets anyway (because netstandard2.0), any particular reason to not just also target net6.0 and net7.0/net8.0, assuming we need new attributes from the newer TFMs? And yeah, for the older TFMs where these aren't available, we can polyfill as usual...

(In other words, I agree it's problematic for net6.0 users to suddenly lose savepoints and stuff)